src/collaborative_platform/apps/dataset_stats/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+from . import views
+
+app_name = 'dataset_stats'
+
+urlpatterns = [
+	path('project/<int:project_id>/versions/', views.versions, name='versions'),
+	path('project/<int:project_id>/version/<str:project_version>/stats/', views.stats, name='stats'),
+    path('project/<int:project_id>/', views.main, name='main'),
```

Comment:
Indentation contains mixed spaces and tabs. What editor allowed that?

Reply:
I believe resolving merge conflicts with Sublime Merge changed it for the urls.py file.